src/pygeoprocessing/geoprocessing.py

```diff
@@ -3900,9 +3898,12 @@ def numpy_array_to_raster(
     Args:
         base_array (numpy.array): a 2d numpy array.
         target_nodata (numeric): nodata value of target array, can be None.
-        pixel_size (tuple): square dimensions (in ``(x, y)``) of pixel.
-        origin (tuple/list): x/y coordinate of the raster origin.
-        projection_wkt (str): target projection in wkt.
+        pixel_size (tuple): square dimensions (in ``(x, y)``) of pixel. Can be
+            None to indicate no stated pixel size.
+        origin (tuple/list): x/y coordinate of the raster origin. Can be None
+            to indicate no stated origin.
+        projection_wkt (str): target projection in wkt.  Can be None to
+            indicate no projection/SRS.
```

Comment:
Much to my surprise, yes!  I do not think this was always the case.  At least with GeoTiffs, you can just not provide these parameters and then you have a TIFF, which works the same way.  It's great for kernels and other things where we either don't have or don't need the spatial information.

Reply:
Interesting! I tried this out and QGIS still identifies it as a geotiff even if the pixel size, origin, and projection are not provided.

Reply:
Though, it does the same with an arbitrary .tiff image.

Reply:
Yes, it's the same driver as far as GDAL is concerned.  I guess it's because GeoTiff is just an extension of the TIFF spec, that a Tiff can be opened as a GeoTiff?